src/registrar/tests/test_views.py

```diff
@@ -1204,7 +1204,7 @@ def test_domain_user_management(self):
         response = self.client.get(
             reverse("domain-users", kwargs={"pk": self.domain.id})
         )
-        self.assertContains(response, "User management")
+        self.assertContains(response, "Domain managers")
```

Comment:
**(non-blocking) suggestion:** I think for any new devs, TestDomainUserManagement, should change to TestDomainMangers so that it is consistent with the title that will now be used

Reply:
Done.